From e03efaef54d0fbf363817ee373a2140f42a259bc Mon Sep 17 00:00:00 2001
From: Jakob Schiotz <schiotz@fysik.dtu.dk>
Date: Thu, 31 Oct 2019 10:54:49 +0100
Subject: [PATCH] Remove unused scipy import leading to crashes with newest
 scipy.

A function in imaging.py imported scipy.interpolate.scipy but did not
use it.  That function is deprecated and has been removed in the
newest scipy.
---
 pyqstem/imaging.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyqstem/imaging.py b/pyqstem/imaging.py
index 531bab5..b9af124 100644
--- a/pyqstem/imaging.py
+++ b/pyqstem/imaging.py
@@ -194,7 +194,7 @@ class CTF(object):
     def radial_plot(self,shape=None,sampling=None,wavelength=None,energy=None,
                     max_freq=2,ax=None,interpolate=True):
 
-        from scipy.interpolate import spline,interp1d
+        from scipy.interpolate import interp1d
 
         if ax is None:
             fig, ax = plt.subplots()
-- 
2.24.0

